templates/auth/lockscreen.html.ep

```diff
@@ -0,0 +1,53 @@
+<div class="lockscreen-wrapper">
+  <!-- User name -->
+  <div class="lockscreen-name">
+    <%= $username // 'invalid session' %>
```

Comment:
This links with the Auth controller constant problem, should be using a constant value